### PR TITLE
Ankitnrathod/tokenwithcertificate

### PR DESCRIPTION
**Pull request by ankitnrathod**
Hi Team,

I have added new policy for generating token using Certificate, let me know if this is good addition and also any comments are welcom

### CODE REVIEW

**Comment by vladvino** [comment]
@maksimkim please review when you get a chance. Thank you @ankitnrathod.

**Comment by ankitnrathod** [comment]
Hi Team,

Can you please review my code and give your valuable comments for completing PR

File: examples/Get OAuth2 access token from AAD using client id and certificate using key vualt manage identity.xml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+<!-- Parameters: scope - a URI encoded scope value -->
+<!-- Parameters: clientId - an id obtained during app registration -->
+<!-- Parameters: KeyVaultName - Key Vault name where we need to get certificate -->
+<!-- Parameters: CertName - Certificate name in Key Vault -->
+<!-- Parameters: cachetime - Cache time in seconds for token -->
+<!-- Copy the following snippet into the inbound section. -->
+
+<policies>
+  <inbound>
+    <cache-lookup-value key="@("bearerToken")" variable-name="bearerToken" />
```
Comment (maksimkim) [inline examples/Get OAuth2 access token from AAD using client id and certificate using key vualt manage identity.xml:16]: does "key" have to be an expression here?

**Review by maksimkim — APPROVED** [review]

File: examples/Get OAuth2 access token from AAD using client id and certificate using key vualt manage identity.xml (hunk 1 above)
Comment (ankitnrathod) [reply examples/Get OAuth2 access token from AAD using client id and certificate using key vualt manage identity.xml:16]: we can remove this,  I was getting token from root policy under APIM

**Comment by ankitnrathod** [comment]
Hi Team,

Can you please help to merge the conflict

**Pull request merged**